libs/dh/wholesale/data-access-api/src/lib/dh-wholesale-data-access-api.store.ts

```diff
@@ -72,6 +68,9 @@ export class DhWholesaleBatchDataAccessApiStore extends ComponentStore<State> {
   loadingBatchesErrorTrigger$: Subject<void> = new Subject();
   loadingBasisDataErrorTrigger$: Subject<void> = new Subject();
 
+  loadingBatchErrorTrigger$: Subject<void> = new Subject();
```

Comment:
nit: Just for readability, perhaps it would be easier to spot that there is a difference between two variables if you moved this to L69

Reply:
Good point! I will change that :)